### DISCUSSION TITLE
Is this recommended to start a new blog (first timer) in Quarto now that R Markdown is superseded ?

### DISCUSSION THREAD

**Discussion by Minh-AnhHuynh** [quarto-dev/quarto-cli#2099] [Q&A] [answered]
Hello,

Is Quarto actually replacing entirely the blog features that you can currently find on HUGO from blogdown using this [guide](https://slides.yihui.org/2022-useR-blogdown.html) ?

Thanks

**Comment by cscheid** [comment]
We have a page with FAQs on quarto/blogdown/rmarkdown: https://quarto.org/docs/faq/rmarkdown.html

**Comment by cderv** [comment, marked as answer]
To add a bit more on the previous answer, Quarto is not replacing **blogdown** which uses Hugo to create blog website, but Quarto does offer features for blog website https://quarto.org/docs/websites/website-blog.html without using Hugo as site generator. 

Regarding equivalence between R Markdown ecosystem and Quarto, you can refer to the specific FAQ that was link 

So in short, as a first timer, it is a matter of preference:

* If you are working mainly in R and like the Hugo templates and framework, starting a new blog using **blogdown** is totally a valid option
* If you are working also in Python and other languages, or just if you are interested to benefit from Quarto features and don't need to use a Hugo template, then Quarto websites with blog feature is a fresh and new good option to start with. 

At the end it is a matter of preference. R Markdown is not going away, and **blogdown** will still exists and be maintained if that was one of your concern. 

Hope it helps

**Reply by harrelfe** [reply]
My hugo-netlify-blogdown blog at fharrell.com is getting hard to maintain as you really can't update hugo styles to new versions, and the blogdown-hugo interaction from the command line (hugo serve site) seems to have broken in part with an update to one of the component apps.  Also the basic style template for articles is too restrictive and I'm writing more of the articles using Quarto and full html rendering.   So I'm second guessing using hugo and wishing that Quarto had been around when I created the blog.

**Reply by cderv** [reply]
That is great feedback @harrelfe ! Thanks for sharing !

**Reply by mcanouil** [reply]
I agree and in addition blogdown + template + HUGO is a headache to upgrade/maintain.
They are not released at the same pace and it's three different layers using different languages/syntax.
With Quarto being here, I would not recommend blogdown at all, and even encourage to make the switch (I did not yet do that myself 😅)

**Reply by cderv** [reply]
> With Quarto being here, I would not recommend blogdown at all, and even encourage to make the switch (I did not yet do that myself 😅)

I think it is meant for different use cases too. Hugo offers a lot of website template with different features - this could be of interested to someone needing one of this theme. Regarding the headache, for websites or blog, I don't think it is like an analysis or other project, it does not need to upgrade always. It is easy to have a hugo website, pin a version of Hugo, and of the template, and then you can have a site working for a long time. **blogdown** is here to help R user leverage that. 
If someone is happy with it, I would not recommend switching. However, if there are limitation encountered, or just someone wants a more scientific oriented stack, then Quarto is definitely a good option. Theming will be limited to Bootstrap though, unless developping its own template, leveraging custom listing page for example. 

At the end, for blogs, it is really a matter of choice I believe :) But yeah... Quarto Website with Blog feature are really good option for scientific oriented blog ! 😄

**Reply by mcanouil** [reply]
My point was mostly for users wanting to compute code and not only static text.
It was not clear in my message 😅

**Reply by Minh-AnhHuynh** [reply]
> My hugo-netlify-blogdown blog at fharrell.com is getting hard to maintain as you really can't update hugo styles to new versions, and the blogdown-hugo interaction from the command line (hugo serve site) seems to have broken in part with an update to one of the component apps. Also the basic style template for articles is too restrictive and I'm writing more of the articles using Quarto and full html rendering. So I'm second guessing using hugo and wishing that Quarto had been around when I created the blog.

Thank you for your feedback. In the end I've found it very easy to begin the first quarto blog draft. I agree that it looks like it can be a nice tool for a scientific blog.
